mergify_engine/service.py

```diff
@@ -40,8 +45,8 @@ def setup(service_name: str, dump_config: bool = True) -> None:
         sentry_sdk.utils.MAX_STRING_LENGTH = 2048
 
     ddtrace.config.version = _version
-    statsd.constant_tags.append(f"service:{service_name}")
-    ddtrace.config.service = service_name
+    statsd.constant_tags.append(f"service:{SERVICE_NAME}")
+    ddtrace.config.service = SERVICE_NAME
```

Comment:
this whole sounds unrelated to this PR?

Reply:
redis-py allow use to set the client name again, so I change that.